components/browser/state/src/main/java/mozilla/components/browser/state/state/BrowserState.kt

```diff
@@ -11,11 +11,14 @@ import mozilla.components.lib.state.State
  *
  * @property tabs the list of open tabs, defaults to an empty list.
  * @property selectedTabId the ID of the currently selected (active) tab.
+ * @property completedDownloads a set of ids of the completed downloads.
  * @property customTabs the list of custom tabs, defaults to an empty list.
  */
 data class BrowserState(
     val tabs: List<TabSessionState> = emptyList(),
     val selectedTabId: String? = null,
 
+    val completedDownloads: Set<Long> = emptySet(),
```

Comment:
Is completed download a browser wide value or a tab specific value?  Will we ever need the tab to show that its downloading/finished downloaded?  This might require more conversation.

Reply:
The current behavior in `AndroidDownloadManager` and `AbstractFetchDownloadService` doesn't seem to contain any tab information, so I think it should go to the BrowserState, however I might be not seeing the big picture, so I'll leave the judgement to people who has more knowledge than me in the code base. 
Let me know what do you think please.

Reply:
I agree, this one will need @pocmo to think about.